### ISSUE TITLE
Unhandled exception upon running 'Disable start menu web search' 

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [5cover/WinClean#3]
When I add the  'Disable start menu web search'  to the list of scripts, I see this:

![image](https://user-images.githubusercontent.com/47056543/199691571-aafd98a4-8b65-423d-9ab1-49044f0ba332.png)

Worse: clicking 'OK' in that window throws further errors.

(I did not choose to create a restore point.)

WinClean 1.2.0

Windows 10 Pro - 21H2 (2009: 19044)

**Comment by 5cover** [comment]
I wasn't able to reproduce your issue. Are you sure you've downloaded the latest version? Former version [1.1.1](https://github.com/5cover/WinClean/releases/tag/v1.1.1) was mistakenly released as 1.2.0.

**Comment by LinuxOnTheDesktop** [comment]
Thanks.

I do have the latest version - I got it from Github just now. (But to avoid confusion, shouldn't you have bumped the version number to 1.2.1?)

I find that, having rebooted, I cannot reproduce the problem. But I can tell you that, when I had the problem, first I executed some of the program's other scripts, and then I added the script of my title, and then (for a second time) I pressed the execute button. That is when the exception occured. The other scripts - the ones I ran first - were, I think, as follows.

![image](https://user-images.githubusercontent.com/47056543/199698506-e197a146-ecda-4426-abeb-a6a13e646392.png)

Also: you might want to add a warning to the option that disables the 'touch keyboard and handwriting service'. For, somewhat notoriously, disabling that service causes problems for the Windows Terminal application.
